surveys/2024-annual-survey/questions.md

```diff
@@ -414,11 +382,33 @@ Type: select all that apply (optional)
 - Achieving structured concurrency with async code
 - Writing executor-agnostic async code
 - Borrow checker not allowing valid code
+
+- (From here the additional one)
+- Runtime performance
```

Comment:
This is also a blanket statement I think it's not very useful. Does it make sense to say "faster!!11" without specifying where/when/what?

Reply:
I think that we can reword everything to be a "challenge" or a "problem" (since we're looking for problems and roadblocks, rather than solutions). So having "slow runtime performance" seems fine to me.